source/mobile/mobile-troubleshoot.rst

```diff
@@ -68,6 +68,20 @@ A server running Linux has this file located in ``/etc/mime.types``. This might
 
 Some distributions also ship without ``mailcap`` which can result in missing or incorrectly configured mime types.
 
+Messages with emojis aren't being sent from the mobile app
```

Comment:
Let's confirm this is mobile only, it does not look like it is

Reply:
Confirmed via @agnivade that this isn't a mobile-specific issue: https://community-daily.mattermost.com/core/pl/7mxqms74ftrdd8jdzuaeuhbggy